components/gantt/state.md

```diff
@@ -0,0 +1,1158 @@
+---
+title: State
+page_title: Gantt - State
+description: Save, load, change the state of the Gantt for Blazor - sorting, filtering and more.
+slug: gantt-state
+tags: telerik,blazor,gantt,state,save,load,layout,get,set,change
+published: True
+position: 15
+---
+
+# Gantt State
```

Comment:
It's good to link to the GanttState API somewhere or even at a few different places. The URL should be `/blazor-ui/api/Telerik.Blazor.Components.GanttState-1`